UI.tsv

```diff
@@ -420,17 +420,17 @@ UI_20150317_000419	Number of Fragments
 UI_20150317_000420	{memo X}Parts 1
 UI_20150317_000421	{memo X}Parts 2
 UI_20150317_000422	{memo X}Parts 3
-UI_20150317_000423	Congratulations! Your level has increased.
+UI_20150317_000423	Congratulations! You have leveled up.
```

Comment:
I actually like the new translation more, "You leveled up" sounds more warm and is more familiar than "Your level number went up"